modules/standard/Map.chpl

```diff
@@ -713,7 +728,7 @@ module Map {
 
     lhs.clear();
     for key in rhs.keys() {
-      lhs.add(key, rhs.getValue(key));
+      lhs.add(key, try! rhs.getValue(key));
```

Comment:
As correct as this may be, I am wondering if there's something we can do better w.r.t this and the following `try!`s. We can

1. Use the sentinel overload: I'd feel less safe given that it can generate incorrect results without any error/warning. But `for key in a.keys do a.getValue(key, sentinel=junk)` feels pretty safe, nonetheless.
2. We can have an internal `getValue`-like overload that can be used in such straightforward uses. It can do without many checks and error generation/propagation.
3. Change these to have `try!` to be in the out of the loop. This can mitigate execution performance worries to some extent.
4. Leave as-is.

Considering the scope of this PR, (4) is good enough. I think (2) is interesting to think about for performance reasons. But I don't think it is this PR's responsibility. If you want, you can do (3), but it is not a must either.

Reply:
Thanks for the suggestions, I'll go with 3.